File: Beecrowd/Beginner/1101 - 1190/1132 - Multiples of 13/C++/1132_Multiplesof13.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int value1, value2, sum = 0;
    cin >> value1 >> value2;

    if (value1 > value2) swap(value1, value2);

    for (int i = value1; i <= value2; i++) {
        if (i % 13 != 0) {
            sum += i;
        }
    }

    cout << sum << endl;
    
    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1132 - Multiples of 13/Python/1132_Multiplesof13.py
value1 = int(input())
value2 = int(input())

if value1 > value2:
    value1, value2 = value2, value1

sum = 0

for i in range(value1, value2 + 1):
    if i % 13 != 0:
        sum += i

print(sum)
